Lib/pathlib.py

```diff
@@ -922,7 +922,8 @@ def relative_to(self, *other):
         cf = self._flavour.casefold_parts
         if (root or drv) if n == 0 else cf(abs_parts[:n]) != cf(to_abs_parts):
             formatted = self._format_parsed_parts(to_drv, to_root, to_parts)
-            raise ValueError("{!r} does not start with {!r}"
+            raise ValueError("{!r} is not in the subpath of {!r}"
+                    " OR one path is relative and the other is absolute."
```

Comment:
Not convinced about the second half of the message - either it's telling me the same as the first half, or it's not telling me anything useful at all.

Perhaps we could expand the "If it's impossible" part in the docs?

Reply:
The second part is not for the impossible cases. It's for cases like "/usr/home/Doc/cpython/libs" and "./Doc/cpython/libs"

I could not find a easy or clean to separate this.

Reply:
Okay, thanks for checking